businesscentral-monitoring/branch-overrides.yaml

```diff
@@ -14,11 +14,11 @@ modules:
     - name: jboss-eap-modules
       git:
         url: https://github.com/jboss-container-images/jboss-eap-modules.git
-        ref: EAP_744_CR2
+        ref: EAP_746_CR2_2
     - name: jboss-eap-7-image
       git:
         url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-        ref: EAP_744_CR2
+        ref: EAP_LP_746_CR1
```

Comment:
What does LP mean?
Does it mean that it isn't released yet?

Reply:
IIRC it stands for the tags created from the legacy branch, @luck3y can confirm it.

Reply:
LP == "layered product", we just use those for tags created in between our releases when you ask for them :)

Reply:
Thanks @luck3y.